### PR TITLE
New Resolver Streamwire

### PR DESCRIPTION
**Pull request by lukkie112**
New resolver streamwire, generic_resolver

### CODE REVIEW

File: lib/resolveurl/plugins/streamwire.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+
+class StreamWireResolver(ResolveGeneric):
+    name = "streamwire"
+    domains = ['streamwire.net']
+    pattern = r'(?://|\.)(streamwire\.net)/(?:embed-)?([0-9a-zA-Z]+)'
+
+    def get_media_url(self, host, media_id):
+        return helpers.get_media_url(self.get_url(host, media_id),
+                                     patterns=[r'''file:\s*"(?P<url>[^"]+)'''],
```
Comment (Gujal00) [inline lib/resolveurl/plugins/streamwire.py:30]: Are you sure about the pattern?
In your earlier PR you had `src:` instead of `file:`

**Review by Gujal00 — CHANGES_REQUESTED** [review]

File: lib/resolveurl/plugins/streamwire.py (hunk 1 above)
Comment (lukkie112) [reply lib/resolveurl/plugins/streamwire.py:30]: I am not sure, maybe i am wrong with file and scr is right

Comment (lukkie112) [reply lib/resolveurl/plugins/streamwire.py:30]: Can You help me please?

Comment (Gujal00) [reply lib/resolveurl/plugins/streamwire.py:30]: Yes, the patterns should be like this for this hoster
```patterns=[r'''src:\s*"(?P<url>[^"]+)''']```
Please update the PR with this change

Comment (lukkie112) [reply lib/resolveurl/plugins/streamwire.py:30]: i am doing a new PR, Thanks

**Pull request closed without merging**